zeebe/broker/src/test/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirectorTest.java

```diff
@@ -320,6 +322,19 @@ public void shouldCloseAllExportersOnClose() throws Exception {
     verify(exporters.get(1), TIMEOUT).close();
   }
 
+  @Test
+  public void shouldCloseAllExportersOnCloseInPassiveMode() throws Exception {
+    // given
+    passiveExporterRule.startExporterDirector(exporterDescriptors);
```

Comment:
:wrench: it may be worth also adding a "mixed" test case where we have:
- 1 exporter active
- 1 exporter passive
- the first exporter registered throws an exception when `close()` is called
and then we check that every `close()` method has been called

Reply:
The "mixed" scenario shouldn't exist, it's the entire `ExporterDirector` which is either in Active or in Passive mode.

Good call on the error handling though, it looks like we don't have any tests for that :+1:

Reply:
I've added  0da020a for this. It's not specific to the Active/Passive mode so I've left that part out.